public/characterCreator.html

```diff
@@ -1 +1,96 @@
-an interactive character creator 
\ No newline at end of file
+an interactive character creator
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <!--<![endif]-->
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <style>
+
+    </style>
+
+    <title></title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="">
+</head>
+
+<body>
+    <!--[if lt IE 7]>
+            <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="#">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+    <form id="characterCreator" name="characterCreator" action="../creator.js" method="POST">
+        <label>Name</label>
+        <input type="text" id="name" name="name"><br><br>
```

Comment:
ids are for javascript and class names are for adding style, you should add descriptive class name(s) to them all

Reply:
So instead, I'll make ```id="stat"``` and then ```class="<something we define later when doing the styling>"```.  And we only need to keep their identifying marks in ```name``` for the submission of the form?

Reply:
Actually, do I even need id?